src/module.c

```diff
@@ -504,6 +510,8 @@ client *moduleAllocTempClient() {
 
     if (moduleTempClientCount > 0) {
         c = moduleTempClients[--moduleTempClientCount];
+        if (moduleTempClientMinCount > moduleTempClientCount)
+            moduleTempClientMinCount = moduleTempClientCount;
```

Comment:
Personal opinion, I think this code is not easy to understand.

Reply:
Yes. That's correct.